search/search_integration_tests/pre_ranker_test.cpp

```diff
@@ -122,6 +113,7 @@ UNIT_CLASS_TEST(PreRankerTest, Smoke)
 
   PreRanker preRanker(m_engine, ranker, pois.size());
   PreRanker::Params params;
+  params.m_viewport = kViewport;
```

Comment:
Set it to zero then.
There are currently two clients to this code: here and in Processor. While you set the radius explicitly in processor, here you rely on default initialization. Using this code for reference may result in an unexpected behaviour.